packages/react/src/internal/components/TextInputWrapper.tsx

```diff
@@ -55,54 +14,34 @@ export type StyledBaseWrapperProps = {
   isInputFocused?: boolean
   monospace?: boolean
   validationStatus?: FormValidationStatus
+  /** @deprecated Use `size` prop instead */
+  variant?: TextInputSizes
+  size?: TextInputSizes
   width?: string
+  className?: string
+  style?: React.CSSProperties
+  onClick?: React.MouseEventHandler
 } & SxProp
 
 export type StyledWrapperProps = {
   hasLeadingVisual?: boolean
   hasTrailingVisual?: boolean
-  /** @deprecated Use `size` prop instead */
-  variant?: TextInputSizes
-  size?: TextInputSizes
 } & StyledBaseWrapperProps
 
-const textInputBasePadding = '12px'
-export const textInputHorizPadding = textInputBasePadding
-
-const renderFocusStyles = (hasTrailingAction: boolean, isInputFocused: boolean) => {
-  if (hasTrailingAction) {
-    return (
-      isInputFocused &&
-      css`
-        border-color: ${get('colors.accent.fg')};
-        outline: 2px solid ${get('colors.accent.fg')};
-        outline-offset: -1px;
-      `
-    )
-  }
-  return css`
-    &:focus-within {
-      border-color: ${get('colors.accent.fg')};
-      outline: 2px solid ${get('colors.accent.fg')};
-      outline-offset: -1px;
-    }
-  `
-}
-
-export const TextInputBaseWrapper = styled.span<StyledBaseWrapperProps>`
-  font-size: ${get('fontSizes.1')};
-  line-height: 20px;
-  color: ${get('colors.fg.default')};
+export const StyledTextInputBaseWrapper = styled.span<StyledBaseWrapperProps>`
+  display: inline-flex;
+  min-height: var(--base-size-32);
```

Comment:
Typically we've still been using the `${get()}` helper for this but I'm curious what you would say @langermank if it's okay for us to just use custom properties now instead of the calls to the theme?

I think the only downside would be if someone is updating these values for the `ThemeProvider` then we aren't reading them anymore which I think would only impact folks outside of dotcom using Primer